### ISSUE TITLE
question about q_loss and alpha_loss

### ISSUE THREAD

**Issue by xxxkxin** [pranz24/pytorch-soft-actor-critic#40]
in sac.update_parameters(), the "qf_loss" was and optimizer are associated with `F.mse_loss(qf1, next_q_value)`, `F.mse_loss(qf2, next_q_value)`, but for policy_loss and alpha_loss, there seems to be no association between loss function and the optimizer. Did i miss something about it?
